doc/integrator/filter.rst.tmpl

```diff
@@ -3,57 +3,16 @@
 Filter (Querier)
 ================
 
-Filterable layers (public)
---------------------------
+Filterable layers
+-----------------
 
-Per default, layers are not filterable.
-If you wish to provide filter functionality, proceed as follows:
-
-* Verify your project vars file to make sure that the functionality ``filterable_layers``
-  is made available in your project template files:
-
-  - If you have a custom definition for ``functionalities.available_in_templates``
-    or ``admin_interface.available_functionalities``, then ensure that the value
-    ``filterable_layers`` is contained in both of these.
-
-  - Alternatively, you can add these two paths in the ``update_paths`` variable to use
-    the definition provided by ``CONST_vars.yaml`` (and keep the inheritance) instead of replacing it.
-
-* If you wish to provide layers for filtering independently of user authorization (in particular,
-  for not-logged-in users), add the layer names as follows to your project vars file:
-
-.. code:: yaml
-
-    functionalities:
-        anonymous:
-            filterable_layers:
-            - my_layer_name_1
-            - my_layer_name_2
+You should add to your role the functionality names ``filterable_layers`` with the layer name as value.
```

Comment:
IMHO this is a little short.

Per default, layers are not filterable.
If you wish to provide filter functionality for some layers, proceed as follows:
- For each layer you want to be "filtrable" create a "functionality" record with "filtrable_layers" as name and the layer name as value.
- In any role (anonymous, registered, intranet or any other role) check the corresponding functionality records.

Reply:
=> https://github.com/camptocamp/c2cgeoportal/pull/6152